core/src/mill/define/Applicative.scala

```diff
@@ -14,14 +14,14 @@ import scala.reflect.macros.blackbox.Context
   * Applier.zipMap(applyable1, applyable2){ (a1, a2, ctx) => ... a1 ... a2 ... }
   */
 object Applicative {
-  trait ApplyHandler[M[+_]]{
+  trait ApplyHandler[M[+ _]] {
     def apply[T](t: M[T]): T
   }
-  object ApplyHandler{
+  object ApplyHandler {
     @compileTimeOnly("Target#apply() can only be used with a T{...} block")
-    implicit def defaultApplyHandler[M[+_]]: ApplyHandler[M] = ???
+    implicit def defaultApplyHandler[M[+ _]]: ApplyHandler[M] = ???
   }
-  trait Applyable[M[+_], +T]{
+  trait Applyable[M[+ _], +T] {
```

Comment:
Any idea why scalafmt wants a space between the `+` and the `_`, but not between `+` and `T`?